sklearn/_config.py

```diff
@@ -7,6 +7,7 @@
     'assume_finite': bool(os.environ.get('SKLEARN_ASSUME_FINITE', False)),
     'working_memory': int(os.environ.get('SKLEARN_WORKING_MEMORY', 1024)),
     'print_changed_only': True,
+    'repr_html': False,
```

Comment:
Should we name this visual_repr?

Reply:
I have no strong feelings for or against `visual_repr`.

(I choose `repr_html` just to match the jupyter naming convention.)

Reply:
Which would you rather see or write at the top of _every_ notebook?

Reply:
`visual_repr` since is sounds nicer to say. lol